Automattic/woocommerce-payments
# feat: update multi-currency settings url

Fixes #

#### Changes proposed in this Pull Request









Just a link update

#### Testing instructions





- Ensure you have `_wcpay_feature_account_overview_task_list` with `'1'` in your `wp_options`
- Ensure the "Enable UPE checkout" flag is enabled
- Ensure the "Enable UPE settings changes" flag is enabled
- Ensure the "Enable Customer multi-currency" flag is enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods
- Link is updated to go to the multi-currency page
![Screen Shot 2021-07-14 at 12 11 33 PM](https://user-images.githubusercontent.com/273592/125664014-e8516417-7462-4625-8ac4-0555cdfa944f.png)


-------------------
